api/inventorypb/agents.proto

```diff
@@ -877,6 +885,10 @@ message AddRDSExporterRequest {
   map<string, string> custom_labels = 5;
   // Skip connection check.
   bool skip_connection_check = 6;
+  // Basic metrics are disabled.
+  bool basic_metrics_disabled = 7;
```

Comment:
Why these variables are different from `disable_basic_metrics` and `disable_enhanced_metrics`?
I think that they should be the same or at least swapped.

Reply:
They should be swapped. `basic_metrics_disabled` describes the current state. `disable_basic_metrics` describes the needed action.